approval_polls/templates/base.html

```diff
@@ -7,8 +7,10 @@
     <meta name='viewport' content='width=device-width, initial-scale=1'>
     <link rel='stylesheet' type='text/css' href='{% static 'vendor/bootstrap/css/bootstrap.min.css' %}'>
     <link rel='stylesheet' type='text/css' href='{% static 'approval_polls/style.css' %}'>
+    <link rel="stylesheet" href='{% static 'vendor/jquery-datetimepicker/css/jquery.datetimepicker.css' %}'>
```

Comment:
I think this should be moved to create.html instead, since this would load every time even when a user is accessing pages other than the create poll page.